app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,14 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+    <div class="container">
+      <%= link_to t("title.header"), "#", id: "logo"%>
+      <nav>
+        <ul class="nav navbar-nav navbar-right">
+          <li><%= link_to t("navbar.home"), "/static_pages/home"%></li>
```

Comment:
`"/static_pages/home"` -> e dùng path helper của Rails route cung cấp nhé
https://guides.rubyonrails.org/routing.html#path-and-url-helpers

Fix tương tự bên dưới